troposphere/static/js/components/identities/detail/resources/IdentityResourcesWrapper.jsx

```diff
@@ -1,7 +1,6 @@
 import React from "react";
 import Backbone from "backbone";
-//FIXME: these imports shouldnt _require_ using ../../.. 
-import SubMenu from "../../../projects/detail/resources/SubMenu";
+
```

Comment:
It is not dead (yet).

`<SubMenu />` was shifted from the wrapper component to `<ButtonBar />` (src: [components/projects/detail/resources/ButtonBar.jsx](https://github.com/cyverse/troposphere/blob/master/troposphere/static/js/components/projects/detail/resources/ButtonBar.jsx)).